.clusterfuzzlite/parse_fuzzer.cpp

```diff
@@ -0,0 +1,27 @@
+
+#include "csv.h"
+#include <unistd.h>
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+  char filename[256];
+  sprintf(filename, "/tmp/libfuzzer.%d.csv", getpid());
+  FILE *fp = fopen(filename, "wb");
+  if (!fp)
+    return 0;
+  fwrite(data, size, 1, fp);
+  fclose(fp);
+
+  io::CSVReader<3> in(filename);
```

Comment:
There is an overload that does not access the file system:
```
LineReader(const char *file_name,
             std::unique_ptr<ByteSourceBase> byte_source)
```
There is a derived source that just takes a buffer:
```
NonOwningStringByteSource(const char *str, long long size)
```
I guess deriving from this class and bypassing OS file system calls is a good idea as it should decrease the running time per test allowing for more tests to fit into the given time budget.

Reply:
Unless you're strongly opinionated on this one I'm happy to leave it as is since I think this entrypoint is good to target and I think performance is okay wrt. exploring the code that is within the fuzzers' reach.